ruby-challenges.rb

```diff
@@ -22,8 +55,25 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+# ----- FUNCTION INFO
+# -- input: string
+# -- output: a string, with all vowels removed
+#
+# ----- STEPS:
+# -- 1. Create a function called delete_vowel which will accept a param of str.
+# -- 2. Store the result of using .delete in a temporary variable to remove all instances of lowercase vowels.
+# -- 3. Print the temporary variable after using .delete a second time to catch all insteances of uppercase vowels.
 
 
+def delete_vowel(str)
+ temp_str = str.delete('aeiou')
+ p temp_str.delete('AEIOU')
```

Comment:
You can add any collection of characters in the delete method and it will remove any of those characters.  So in a sense, you could combine lines 69 and 70 into one line.